tests/utils.py

```diff
@@ -165,8 +165,8 @@ def cmp_exprs(left, right):
     return False
 
 
-def assert_code_equal(actual, expected):
-    assert actual.strip() == expected.strip()
+def assert_code_equal(actual, *expected):
+    assert any(actual.strip() == expect.strip() for expect in expected)
```

Comment:
I'd suggest to keep the existing function as is and introduce a new one:
```
def assert_code_equal_any_of(...)
```
This way the semantics of the check will be clear vs now the logic is obscured unless you read the function's body.